front/types/user.ts

```diff
@@ -77,7 +78,8 @@ export type UserProviderType =
   | "okta"
   | "samlp"
   | "waad"
-  | null;
+  | null
+  | DirectoryType;
```

Comment:
let's keep null when coming from proivisioning (this is supposed to have the  authentication provider name )